src/core/tiledscene/qgstiledscenelayerrenderer.cpp

```diff
@@ -261,6 +262,15 @@ bool QgsTiledSceneLayerRenderer::renderTiles( QgsTiledSceneRenderContext &contex
       painter->setPen( pen );
       painter->setBrush( brush );
       painter->drawPolygon( tile.boundary );
+#if 1
```

Comment:
is this intentional?

Reply:
yeah -- the same is in the vector tiles code. I just wanted to make it easy to disable if we don't want this behavior in future.